facade/service.go

```diff
@@ -1180,7 +1181,9 @@ func (f *Facade) ScheduleService(ctx datastore.Context, serviceID string, autoLa
 	mutex := getTenantLock(tenantID)
 	mutex.RLock()
 	defer mutex.RUnlock()
-	return f.scheduleService(ctx, tenantID, serviceID, autoLaunch, synchronous, desiredState, false, false)
+	// Pause requests are always expedited
+	emergency := desiredState == service.SVCPause
+	return f.scheduleService(ctx, tenantID, serviceID, autoLaunch, synchronous, desiredState, false, emergency)
 }
```

Comment:
I don't think we need to make Pause an emergency anymore since when we Snapshot, we're calling lockTenant, which calls Wait, which waits until there is nothing left in the queue.  (probably no negative side effects though)